src/main/java/org/mitre/synthea/export/FhirStu3.java

```diff
@@ -983,10 +1002,10 @@ private static Type mapValueToFHIRType(Object value, String unit) {
 
     } else if (value instanceof Condition) {
       Code conditionCode = ((HealthRecord.Entry) value).codes.get(0);
-      return mapCodeToCodeableConcept(conditionCode, SNOMED_URI);
+      return mapCodeToCodeableConcept(conditionCode, conditionCode.system);
```

Comment:
Changing all these to take in `code.system` will work; I think that plus the changes to the method below will result in identical output to how it works today. But conceptually it feels wrong to me - the intent of providing a separate `system` parameter to the method was that it would support using a different system than what was already in `code.system`. With this change, everywhere it's called it gets passed in the value from `code.system` so the extra parameter isn't needed. 
Can you confirm that's the case and if so, remove the extra parameter from the method and its doco so that it just uses the `from.system` implicitly without having to pass it in? There may someday be a case in the future that we want to provide a different system, but if it's not used we can just clean it up.

Reply:
Check this carefully...